lib/install/is-fs-access-available.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+var fs = require('fs')
+var process = require('process')
+var semver = require('semver')
+
+var isWindows = process.platform === 'win32'
+
+// fs.access first introduced in node 0.12 / io.js
+if (!fs.access) {
+  module.exports = false
+} else if (!isWindows) {
+  // fs.access always works on non-Windows OSes
+  module.exports = true
+} else {
+  // The Windows implementation of `fs.access` has a bug where it will
+  // sometimes return access errors all the time for directories, even
```

Comment:
> Under some circumstance that I don't understand, it always claims directories to be unwritable. These circumstances don't show up all the time, but when they do, they always result in failures.

from trial and error: if directories would require admin access they are reported as unwritable even if running as admin